BUILD/mdbci/build.sh

```diff
@@ -122,11 +118,7 @@ if [ "$try_already_running" == "yes" ] ; then
 fi
 if [[ "$do_not_destroy_vm" != "yes" && "$try_already_running" != "yes" ]] ; then
   echo "Destroying VM"
-  vagrant destroy -f
-  cd ..
-  rm -rf $name
-  rm -rf ${name}.json
-  rm -rf ${name}_netwotk_config
+  ${mdbci_dir}/mdbci destroy $name
```

Comment:
Same comment here. You use relative path to the configuration, not the absolute path.

Here you have also deleted the change of the working directory, so the code won't work.

The same issue of calling destroy with the relative path to the configuration can be seen in all